http/generators.http

```diff
@@ -7,4 +7,13 @@ GET {{host}}/{{generator}}/lot-pdf-result
 Content-Type: application/json
 
 ### generate laptopItems for one already exist lotPDFResult
-GET {{host}}/{{generator}}/laptop-item/{{lotPDFResult}}
\ No newline at end of file
+GET {{host}}/{{generator}}/laptop-item/{{lotPDFResult}}
+
+### receive all lots from prozorro api
+GET {{host}}/api_lot_extract_all
+Content-Type: application/json
+
+### receive one lot from prozorro api
+GET {{host}}//:lotId
```

Comment:
why it's 2 slashes and everywhere else is one?

Reply:
It's a mistake. Done.